aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -225,6 +233,9 @@ bool check_mem_efficient_hardware_support(sdp_params const& params, bool debug)
       }
       return false;
   }
```

Comment:
@pruthvistony Isn't this where we would need to add a return true to address the "Control reaching end of non-void function" error?

Reply:
No it is NOT required here